src/Components/ErrorBanner/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './styles.module.scss';
+
+export const ErrorBanner = ({ errorText }) => {
+  return (
+    <div className={styles.ErrorBanner}>
+      <span>{errorText}</span>
```

Comment:
I think you can use just a `message` prop, it's already rendered by `Error` component, `errorMessage / errorText`  is an overhead